FreeRTOS/Test/CBMC/proofs/Queue/QueueGenericCreateStatic/QueueGenericCreateStatic_harness.c

```diff
@@ -52,8 +52,10 @@ void harness(){
     __CPROVER_assume(uxQueueLength > ( UBaseType_t ) 0);
 
     // QueueGenericCreateStatic asserts the following equivalence
-    __CPROVER_assume( ( pucQueueStorage && uxItemSize ) ||
-                      ( !pucQueueStorage && !uxItemSize ) );
+    /* A queue storage area should be provided if the item size is not 0, and
```

Comment:
This may help : https://en.wikipedia.org/wiki/De_Morgan%27s_laws

Reply:
oops. Yes I did use demorgans to evaluate a few transformations but clearly not enough. With more transformations, they are logically equal. My mistake, preemptively assumed inequality after getting about halfway through the necessary transformations.

Though, I'd argue it's still better to change them because it requires reader to do several boolean transformations to verify it matches the code